CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Change Log
 
+## 0.1.1 - 2020-06-10
+
+### Added
+- Display Function triggers to tree
+
+### Changed
+- Visual Studio Code - Insiders is no longer a requirement
+- Azure Functions extension is no longer a direct dependency
```

Comment:
I would mention how you changed the default app folder. If CTI was confused by it, I could see other users confused, too